linux_os/guide/system/permissions/files/permissions_local_var_log/rule.yml

```diff
@@ -56,8 +56,10 @@ template:
     name: file_permissions
     vars:
         excluded_files@sle15: ['*[bw]tmp', '*lastlog']
+        excluded_files@ubuntu2004: ['*[bw]tmp', '*lastlog']
```

Comment:
please include to this list: 'history.log' and 'eipp.log.xz' 
both are needed for apt and have their permissions reset when running apt.
Also please include the same for ubuntu2204 and remove the bash remediation.

Reply:
Thanks for the catch!